projects/plugins/jetpack/extensions/blocks/ai-assistant/lib/prompt/index.ts

```diff
@@ -224,6 +231,55 @@ function getTonePrompt( {
 	];
 }
 
+function getJetpackFormCustomPrompt( {
+	content,
+	role = 'user',
+	request,
+}: PromptOptionsProps ): Array< PromptItemProps > {
+	if ( ! request ) {
+		throw new Error( 'You must provide a custom prompt for the Jetpack Form Custom Prompt' );
+	}
+
+	return [
+		{
+			role: 'system',
+			content: `You are an expert developer in Gutenberg, the WordPress block editor, and thoroughly familiar with the Jetpack Form feature. Your content will be used inside a Jetpack Form block that already exists.
+Writing rules:
+- Execute the request without any acknowledgment or explanation to the user.
+- If you cannot generate a meaningful response to a user's request, reply with “__JETPACK_AI_ERROR__“. This term should only be used in this context, it is used to generate user facing errors.
+- Avoid sensitive or controversial topics and ensure your responses are grammatically correct and coherent.
+- DO NOT add any addtional feedback to the user, just generate the requested block structure and nothing else.`,
+		},
+		{
+			role,
+			content: `Handle the following request, delimited with ${ delimiter }: ${ getDelimitedContent(
```

Comment:
I'm not sure about the delimiter. Did you test how it behaves when composing forms?

Reply:
I tested with the PoC branch, it looks very well. Let's pay attention to how it processes the response.